tensorflow/src/main/scala/com/spotify/featran/tensorflow/package.scala

```diff
@@ -70,11 +70,8 @@ package object tensorflow {
     override def newBuilder: FeatureBuilder[tf.Example] = TensorFlowFeatureBuilder()
   }
 
-  /** [[FeatureBuilder]] for output as TensorFlow `Example` type. */
-  implicit def tensorFlowFeatureBuilder: FeatureBuilder[tf.Example] = TensorFlowFeatureBuilder()
-
-  implicit val exampleFlatReader: FlatReader[tf.Example] = new FlatReader[tf.Example] {
-    import TensorFlowType._
```

Comment:
So we were actually not using the shapeless based `TensorFlowType[T]` converters, just implicits/utilities from that package?

Reply:
yea, basically just utility methods from that package.